pkgs/applications/networking/browsers/firefox/packages.nix

```diff
@@ -116,89 +58,25 @@ rec {
     updateScript = callPackage ./update.nix {
       attrPath = "firefox-esr-68-unwrapped";
       versionSuffix = "esr";
-      versionKey = "ffversion";
-    };
-  };
-
-} // (let
-
-  iccommon = args: common (args // {
-    pname = "icecat";
-    isIceCatLike = true;
-
-    meta = (args.meta or {}) // {
-      description = "The GNU version of the Firefox web browser";
-      longDescription = ''
-        GNUzilla is the GNU version of the Mozilla suite, and GNU
-        IceCat is the GNU version of the Firefox web browser.
-
-        Notable differences from mainline Firefox:
-
-        - entirely free software, no non-free plugins, addons,
-          artwork,
-        - no telemetry, no "studies",
-        - sane privacy and security defaults (for instance, unlike
-          Firefox, IceCat does _zero_ network requests on startup by
-          default, which means that with IceCat you won't need to
-          unplug your Ethernet cable each time you want to create a
-          new browser profile without announcing that action to a
-          bunch of data-hungry corporations),
-        - all essential privacy and security settings can be
-          configured directly from the main screen,
-        - optional first party isolation (like TorBrowser),
-        - comes with HTTPS Everywhere (like TorBrowser), Tor Browser
-          Button (like TorBrowser Bundle), LibreJS, and SpyBlock
-          plugins out of the box.
-
-        This package can be installed together with Firefox and
-        TorBrowser, it will use distinct binary names and profile
-        directories.
-      '';
-      homepage = "https://www.gnu.org/software/gnuzilla/";
-      platforms = lib.platforms.unix;
-      license = with lib.licenses; [ mpl20 gpl3Plus ];
-    };
-  });
-
-in {
-
-  icecat = iccommon rec {
-    ffversion = "60.3.0";
-    icversion = "${ffversion}-gnu1";
-
-    src = fetchurl {
-      url = "mirror://gnu/gnuzilla/${ffversion}/icecat-${icversion}.tar.bz2";
-      sha256 = "0icnl64nxcyf7dprpdpygxhabsvyhps8c3ixysj9bcdlj9q34ib1";
-    };
-
-    patches = [
-      ./no-buildconfig.patch
-      missing-documentation-patch
-    ];
-    meta.knownVulnerabilities = [ "Support ended around October 2019." ];
-  };
-
-  # Similarly to firefox-esr-52 above.
-  icecat-52 = iccommon rec {
-    ffversion = "52.6.0";
-    icversion = "${ffversion}-gnu1";
-
-    src = fetchurl {
-      url = "mirror://gnu/gnuzilla/${ffversion}/icecat-${icversion}.tar.bz2";
-      sha256 = "09fn54glqg1aa93hnz5zdcy07cps09dbni2b4200azh6nang630a";
+      versionKey = "version";
     };
-
-    patches = [
-      # this one is actually an omnipresent bug
-      # https://bugzilla.mozilla.org/show_bug.cgi?id=1444519
-      ./fix-pa-context-connect-retval.patch
-    ];
-
-    meta.knownVulnerabilities = [ "Support ended in August 2018." ];
   };
+} // lib.optionalAttrs (config.allowAliases or true) {
+  #### ALIASES
+  #### remove after 20.03 branchoff
```

Comment:
I'm almost certain we'll forget this :rofl: Can you open an issue and add it to the miltestone?
(this reminds me we need to clean up `aliases.nix`)